src/components/ArticleList.jsx

```diff
@@ -0,0 +1,12 @@
+import Article from './Articles';
+
+const ArticleList = ({isLoading,articles}) => {
+  return (<>
+        { isLoading ? <h1>Loading...</h1> :
```

Comment:
This would be cleaner if you handle the isLoading in the parent component and only render the child if isLoading isn't true and you have data